### PR TITLE
Add accessibility labels for chat long press

### LINKED ISSUE #14670: No accessibility labels for items in the chat long-press menu
# Bug Report

## Problem 

Please, kindly add accessibility IDs for all items in the long-press menu:

- View profile
- Edit nickname
- Mark as read
- Mute chat
- Notifications
- Pinned messages
- Show QR code
- Share profile
- Clear history
- Delete chat

Screenshot 2022-12-30 at 10 45 00

### Reproduction

1. Create a 1-1 chat
2. Go to the home screen
3. Long tap the chat to call the menu

Also, please add accessibility IDs for these two buttons (Long press on 1-1 chat > `Delete chat`):

Screenshot 2022-12-30 at 12 34 31

### Additional Information

- Status version: nightly 30/12
- Operating System: Android, iOS

[comment]: # (Please, add logs/notes if necessary)

### PR DESCRIPTION
**Pull request by ibrkhalil**
fixes #14670 
status: ready

### CODE REVIEW

**Review by jo-mut — APPROVED** [review]

File: src/status_im2/common/home/actions/view.cljs [hunk 1]
```diff
@@ -54,294 +54,323 @@
...
-          :chevron?  false}))
+  (entry {:icon                :i/qr-code
+          :label               (i18n/label :t/show-qr)
+          :on-press            #(js/alert "TODO: to be implemented, requires design input")
+          :danger?             false
+          :accessibility-label :show-qr-code
+          :sub-label           nil
+          :chevron?            false}))
 
 ;; TODO(OmarBasem): to be implemented.
```
Comment (J-Son89) [inline src/status_im2/common/home/actions/view.cljs:265]: @OmarBasem, is there an issue for this TODO tracked?
Seems there could easily be a github issue - "get design input for actions in common home actions"

Comment (J-Son89) [reply src/status_im2/common/home/actions/view.cljs:265]: same for the on-press.. maybe we can add an issue for the ones which are not yet completed?

**Comment by churik** [comment]
@ibrkhalil 
unfortunately no accessibility-labels on required elements were added
Appium 2023-01-03 13-19-55

Tested on build with https://github.com/status-im/status-mobile/commit/5ce9e51908e05c22aa6f0d1406c2b260506cba5b commit

File: src/status_im2/common/home/actions/view.cljs (hunk 1 above)
Comment (OmarBasem) [reply src/status_im2/common/home/actions/view.cljs:265]: @J-Son89 There isn't an issue. Will create an issue now

**Review by OmarBasem — APPROVED** [review]

**Comment by churik** [comment]
thanks, it is working. Can be merged

**Pull request merged**